Does not use ocamlfind, so this fails on nums.cmxa.

--- a/scripts/mk_project.py
+++ b/scripts/mk_project.py
@@ -101,7 +101,6 @@
     add_c_example('maxsat')
     add_dotnet_example('dotnet_example', 'dotnet')
     add_java_example('java_example', 'java')
-    add_ml_example('ml_example', 'ml')
     add_z3py_example('py_example', 'python')
     return API_files
 
